Plugins/GitFlow/GitFlowForm.cs

```diff
@@ -337,9 +351,37 @@ private bool RunCommand(ArgumentString commandText)
 
             txtResult.Text = resultText;
 
+            ApplyCommandToAllSubmodules(commandText);
+
             return result.ExitCode == 0;
         }
 
+        private void RunOutputCommand(ArgumentString commandText)
```

Comment:
This is mostly a duplication of RunCommand()
Can this be merged?

Reply:
I don't recommend merging it since the method does seperate things. The RunCommand method doesn't have a result code for example. If duplicate code must be avoided then I suggest putting all the text and drawing actions to a seperate method.